bcipy/signal/model/offline_analysis.py

```diff
@@ -92,13 +92,16 @@ def offline_analysis(data_folder: str = None,
     # read_data_csv already removes the timespamp column.
     channel_map = analysis_channels(channels, type_amp)
 
-    x, y, _, _ = trial_reshaper(t_t_i, t_i, data,
-                                mode=mode, fs=fs, k=downsample_rate,
-                                offset=offset,
-                                channel_map=channel_map,
-                                trial_length=trial_length)
-
     model = PcaRdaKdeModel(k_folds=k_folds)
+    x, y = data_reshaper(input_data_type=model.input_data_type,
+                         trial_target_info=t_t_i,
+                         timing_info=t_i,
+                         eeg_data=data,
+                         fs=fs,
+                         trials_per_inquiry=parameters.get('stim_length', 10),
```

Comment:
I don't think we should use defaults in offline analysis if we can avoid it (especially for important things like stim_length). I'd rather we fail.